components/FilterBar/FilterBar.js

```diff
@@ -24,7 +24,7 @@ const FilterBar = (props) => (
       </Link>
     </div>
     <div className={s.filterOption}>
-      <Link to="/locations" queryString={getNewSortQueryString(props.queryString, !props.sortDist)}>
+      <Link to="/locations" replaceState={true} queryString={getNewSortQueryString(props.queryString, !props.sortDist)}>
```

Comment:
you can just add the prop with no value, `replaceState`. React will assume the value is true then.